### PR TITLE
Enable some features to improve the programming experience

### PR DESCRIPTION
**Pull request by THEb0nny**
Enable addNewTypeScriptFile so that you can split your code into multiple files. Enable blocksCollapsing to collapse blocks for convenience. Turn off highContrast it doesn't work now. We'll see later...

All enabled options work.

### CODE REVIEW

**Comment by THEb0nny** [comment]
Also added the errorList option. Works.

**Comment by THEb0nny** [comment]
@jwunderl please see the changes.

File: pxtarget.json [hunk 1]
```diff
@@ -109,6 +109,8 @@
         "copyrightText": "LEGO, the LEGO logo, MINDSTORMS and the MINDSTORMS EV3 logo are trademarks and/ or copyrights of the LEGO Group. ©2018 The LEGO Group. All rights reserved.",
         "crowdinProject": "makecode",
         "selectLanguage": true,
+        "blocksCollapsing": true,
+        "highContrast": false,
```
Comment (jwunderl) [inline pxtarget.json:113]:
```suggestion
        "highContrast": true,
```
Drop this one, needs to stay in; I'll try and find some time to port over a fix for whatever is causing it to do nothing, probably something simple

**Review by jwunderl — APPROVED** [review]
lgtm besides the high contrast chnage~

**Comment by THEb0nny** [comment]
> lgtm besides the high contrast chnage~

okay

File: pxtarget.json (hunk 1 above)
Comment (THEb0nny) [reply pxtarget.json]: okay

**Comment by THEb0nny** [comment]
@jwunderl please see changes

**Comment by jwunderl** [comment]
Thanks!

**Pull request merged**